File: Apps/DriveSwitch/app/src/main/java/com/driveswitch/CreditCard.java
package com.driveswitch;

/**
 * Created by Joes-Acer on 1/18/2017.
 */

public class CreditCard
{
    public CreditCard()
    {
        _id = Constants.EmptyObjectId;
        _t = "CreditCard";

        FullName = "";
        CardTypeId = Constants.CreditCard.Unknown[0];
        CardTypeName = Constants.CreditCard.Unknown[1];
        Number = "";
        Expires = "";
        Zipcode = "";
        CVVCode = "";
    }
    public String _id;
    public String _t;
    public String FullName;
    public String CardTypeId;
    public String CardTypeName;
    public String Number;
    public String Expires;
    public String Zipcode;
    public String CVVCode;
}

File: WebApi/WebApi/Models/CreditCard.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Web;

using MongoDB;
using MongoDB.Bson;
using MongoDB.Driver;

namespace WebApi.Models
{
    public class CreditCard
    {
        public CreditCard()
        {
            _id = ObjectId.GenerateNewId();
            _t = "CreditCard";

            FullName = "";
            CardTypeId = ObjectId.Empty;
            CardTypeName = "";
            Number = "";
            Expires = "";
            Zipcode = "";
            CVVCode = "";
        }

        public ObjectId _id { get; set; }
        public string _t { get; set; }
        public String FullName { get; set; }
        public ObjectId CardTypeId { get; set; }
        public String CardTypeName { get; set; }
        public String Number { get; set; }
        public String Expires { get; set; }
        public String Zipcode { get; set; }
        public String CVVCode { get; set; }
    }
}